File: src/main/java/com/shinowit/entity/OutStock.java
package com.shinowit.entity;

import java.io.Serializable;
import java.util.Date;

public class OutStock implements Serializable {
    private Integer outStockId;

    private Integer userId;

    private Date outTime;

    private String handlerName;

    private Byte outType;

    private Float totalMoney;

    private String remark;

    private static final long serialVersionUID = 1L;

    public Integer getOutStockId() {
        return outStockId;
    }

    public void setOutStockId(Integer outStockId) {
        this.outStockId = outStockId;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public Date getOutTime() {
        return outTime;
    }

    public void setOutTime(Date outTime) {
        this.outTime = outTime;
    }

    public String getHandlerName() {
        return handlerName;
    }

    public void setHandlerName(String handlerName) {
        this.handlerName = handlerName;
    }

    public Byte getOutType() {
        return outType;
    }

    public void setOutType(Byte outType) {
        this.outType = outType;
    }

    public Float getTotalMoney() {
        return totalMoney;
    }

    public void setTotalMoney(Float totalMoney) {
        this.totalMoney = totalMoney;
    }

    public String getRemark() {
        return remark;
    }

    public void setRemark(String remark) {
        this.remark = remark;
    }
}

File: web/js/shinow/app/model/OutStock.js
/**
 * <p>
 * 出库信息表
 * </p>
 * 
 * @author 代码生成器
 * @class app.model.OutStock
 * @extends Ext.data.Model
 */
Ext.define('app.model.OutStock', {
		extend : 'Ext.data.Model',
		alias : 'model.OutStock',
		idProperty:'outStockId',
		fields : [
				{
					name : 'outStockId',
					type: 'int'
				},
				{
					name : 'userId',
					type: 'int'
				},
				{
					name : 'outTime',
					type: 'date'
				},
				{
					name : 'handlerName',
					type: 'string'
				},
				{
					name : 'outType',
					type: 'int'
				},
				{
					name : 'totalMoney',
					type: 'string'
				},
				{
					name : 'remark',
					type: 'string'
				}
		]
});